### ISSUE TITLE
Page title on /privacy should be h1

### ISSUE THREAD

**Issue by CrispinF** [cloudscribe/cloudscribe#762]
Currently h2.

**Comment by CrispinF** [comment]
Most accessibility and SEO recommendations suggest each page should have its own unique h1 heading. Many of our inbuilt views like /privacy use h2 or lower headings, based on a presumption the main website name will be displayed as h1.
Simple solution: make all these h1.
However, there's nothing wrong with having a main site title as h1 and pages within it being h2. The smaller the site, the more appropriate this is. So there's a case for making the page titles in cloudscribe Core configurable, with h1 as the default, and an app setting or content setting determining the page heading element. This could equally apply to SimpleContent pages.
